File: Leetcode/27 - Remove Element/solution.py
def removeElement(nums: List[int], val: int) -> int:
    low, high = 0, len(nums) - 1

    while low <= high:
        if nums[low] == val:
            nums[low] = nums[high]
            high -= 1
        else:
            low += 1

    return low

File: Leetcode/27 - Remove Element/solution.cpp
int removeElement(vector<int>& nums, int val) {
    int low = 0, high = nums.size() - 1;

    while (low <= high) {
        if (nums[low] == val) {
            nums[low] = nums[high];
            high--;
        } else {
            low++;
        }
    }

    return low;
}